the-collab-lab/tcl-16-smart-shopping-list
# 16. As a user, I want the home page to look welcoming and professional

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description



Added styles for the homepage based on the mockups. Also removed unused css / refactored some css into mixins. 

## Related Issue


closes #35 

## Acceptance Criteria
Home page should be styled per mockup using colours and fonts as indicated in our scss file



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|  ✓   | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before


![Screen Shot 2021-02-24 at 9 09 45 PM](https://user-images.githubusercontent.com/46036289/109092514-abb02180-76e4-11eb-81c2-c254d83879af.png)

### After


![Screen Shot 2021-02-24 at 9 10 59 PM](https://user-images.githubusercontent.com/46036289/109092597-d69a7580-76e4-11eb-9476-5f5d9b3b4ceb.png)

## Testing Steps / QA Criteria


1. switch to branch `mr-home-page-ui`
2. npm start
3. to get to homepage, make sure you clear your local storage 
4. check that homepage matches design mockup
